crates/bevy_transform/src/propagate.rs

```diff
@@ -0,0 +1,25 @@
+use bevy_hierarchy::Propagatable;
+
+use crate::{GlobalTransform, Transform};
+
+impl Propagatable for Transform {
```

Comment:
This impl should go in components/mod.rs IMO

Reply:
Yes I also think `Propagate` impl should be next to the components.